frontend/frontend/__main__.py

```diff
@@ -3,6 +3,8 @@
 
 import frontend.protocols.ssh as ssh
 
-key = paramiko.RSAKey(filename="./host.key")
-s = ssh.ConnectionManager(host_key=key, port=2222)
-s.listen()
+if __name__ == '__main__':
+    key = paramiko.RSAKey(filename="./host.key")
+    s = ssh.ConnectionManager(host_key=key, port=2222)
```

Comment:
You may want to parametrize these later on, if you need to go down that rabbit hole, consider https://docs.python.org/3.9/library/argparse.html or one of the many libraries for that xD https://codeburst.io/building-beautiful-command-line-interfaces-with-python-26c7e1bb54df

Reply:
Good idea, we're thinking about creating a config file where we can set these types of settings